src/components/Footer/TodoFooter.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-param-reassign */
+import { useAppContextContainer } from '../../context/AppContext';
+import { footerOption } from './TodoFooter.data';
+import TodoFooterOption from './TodoFooterOption';
+
+const TodoFooter = () => {
+  const { changeFilterType, filterType, todos, deleteCompleted } =
```

Comment:
when here more than 3 values, split it into rows

Reply:
Pretier is not give me do that =)